README.md

```diff
@@ -146,11 +147,11 @@ Custom loaders can be implemented.
 
 ### Extension loading
 
-Additionally, every Vulkan extension has to be loaded explicitly. You can find all extensions under [`ash::extensions`](https://github.com/ash-rs/ash/tree/master/ash/src/extensions).
+Additionally, every Vulkan extension has to be loaded explicitly. You can find all extensions directly under [`ash::*`](https://github.com/ash-rs/ash/tree/master/ash/src/extensions) in a module with their prefix (e.g. `khr` or `ext`).
```

Comment:
Does this link still make sense? Maybe unlink for now, and add a link to the docs.rs page once it exists?

Reply:
Dropped entirely, there's not a single place where extensions are found (and the documentation root will be filled with everything)